test/e2e/playwright/swap/pageObjects/swap-page.ts

```diff
@@ -98,9 +139,25 @@ export class SwapPage {
     await this.page.waitForSelector(`text=/New quotes in 0:${second}/`);
   }
 
-  async waitForTransactionToComplete() {
-    await this.page.waitForSelector('text=/Transaction complete/');
-    await this.closeButton.click(); // Close button
+  async waitForTransactionToComplete(options: { seconds: number }) {
+    let countSecond = options.seconds;
+    let trasnsactionCompleted;
+    do {
+      trasnsactionCompleted = await this.page.$('text=/Transaction complete/');
```

Comment:
Typo in variable name `trasnsactionCompleted`, it should be `transactionCompleted`.

Reply:
Done